src/store/analyze.rs

```diff
@@ -17,7 +21,7 @@ pub struct Match<'a> {
     pub score: f32,
     /// The name of the closest matching license in the `Store`. This will
     /// always be something that exists in the store, regardless of the score.
-    pub name: String,
```

Comment:
Technically a breaking change, but one I think I'm OK with as this is still pre-1.0 and it'd be great to have the "correct" types going forward.

Reply:
Cool, yah, I figured it was ok since it only requires minor changes in calling code.